setup.py

```diff
@@ -7,7 +7,7 @@
     from distutils.core import setup
 
 
-version = '0.1.1'
+version = '0.1.1+optel1'
```

Comment:
we should keep the same nomenclature as the fork django-pyodbc-azure, ie. `0.1.1+optel.1`